modules/microshift-lvms-configuring.adoc

```diff
@@ -3,14 +3,15 @@
 // * microshift_storage/microshift-storage-plugin-overview.adoc
 
 :_content-type: CONCEPT
-[id="lvms-configuring_{context}"]
+[id="microshift-lvms-configuring_{context}"]
 = Configuring the LVMS
 
-{product-title} supports passing through your LVM configuration and allows you to specify custom volume groups, thin volume provisioning parameters, and reserved unallocated volume group space. You can edit the LVMS configuration file at any time. You must restart {product-title} to deploy configuration changes after editing the file.
+{product-title} supports passing through your LVM configuration and allows you to specify custom volume groups, thin volume provisioning parameters, and reserved unallocated volume group space. You can edit the LVMS configuration file you created at any time. You must restart {product-title} to deploy configuration changes after editing the file.
```

Comment:
@ShaunaDiaz Can we add link for "How to restart microshift service" section if there is any?

Reply:
OK thanks, as discussed may be we can add it later.